src/components/FAB/FAB.tsx

```diff
@@ -32,7 +32,7 @@ type Props = $RemoveChildren<typeof Surface> & {
    */
   small?: boolean;
   /**
-   * Custom color for the `FAB`.
+   * Custom text color for the `FAB`.
```

Comment:
Thanks for the PR. Can you tweak the description so it's obvious that it changes the icon and label color?

```suggestion
   * Custom color for the icon and label of the `FAB`.
```